src/main/java/carpetextra/mixins/DispenserBehaviorMobHeadMixin.java

```diff
@@ -1,19 +1,17 @@
 package carpetextra.mixins;
 
-import org.spongepowered.asm.mixin.Mixin;
-import org.spongepowered.asm.mixin.injection.At;
-import org.spongepowered.asm.mixin.injection.Inject;
-import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
-
 import carpetextra.utils.PlaceBlockDispenserBehavior;
 import net.minecraft.block.dispenser.FallibleItemDispenserBehavior;
 import net.minecraft.item.BlockItem;
 import net.minecraft.item.ItemStack;
 import net.minecraft.util.math.BlockPointer;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
 
-@Mixin(targets = {"net/minecraft/block/dispenser/DispenserBehavior$14", "net/minecraft/block/dispenser/DispenserBehavior$15"})
```

Comment:
I was trying to do this port as well but you are way more advanced, but when I was doing it I had $14->$3 and $15->$4 but you have only $3

Reply:
I just assumed Mojang changed the original code, I'll check again and report here 😄

Reply:
On the latest 1c54c8cb commit `$14` and `$15` were mixing into dispensing glowstone and glass bottles, that's weird. Looked into the new code — the right bytecode for mob heads should be `$11` for mob heads and `$12` for wither skeleton skulls, `$3` is a saddle :smile: 

I guess I'm gonna have to just check whether if it works in game :grin:

Reply:
The `$3` is the one used with heads and skeleton skulls, I did miss the `$4` for wither skeleton skull. Thanks for your attention :)

Could not quite figure out why it worked anyway, found out the carpet dispenser was dispensing wither skulls :smile: Going to recheck all the targets and fix it.

Reply:
Fixed with recent commits